### PR TITLE
Prepared input files for DLK

### PR DESCRIPTION
**Pull request by tlhr**
## Input submission checklist

* [x] created a directory for each system based on the [template directory][templates]
* [x] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [x] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [x] added a PDB file with the protein named `protein.pdb`
* [x] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [x] did not re-protonate the residues, but left protonation state of the protein unchanged
* [x] kept cystallographic waters and metals in the PDB file
* [x] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [x] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Summary of changes

* Removed ligands 4neu and 17neu
* Protein termini were capped, and NMA renamed to NME
* Residues ASN241, PHE270, SER326, and CYS335 contained alternative conformations, the default one was used
* The system had no crystallographic waters, metal ions, or other cofactors

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/dlk/ligands.sdf
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/dlk/ligands.sdf:1]: For this study, we decided to only use a single protonation state per ligand, choosing the protonation state that had been used in the original studies by Chen et al. (https://pubs.acs.org/doi/10.1021/acs.jctc.8b00825). In this case, for ligand 4 and 17 we would only include the positively charged amine. Could you remove the ligands 4neu and 17neu from this file? Thank you!

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thanks, @tlhr this looks great! Just the one comment for the ligands file! If you remove the alternate protonation states, could you then also update the preparation instructions accordingly? Thank you!

**Comment by tlhr** [comment]
> Thanks, @tlhr this looks great! Just the one comment for the ligands file! If you remove the alternate protonation states, could you then also update the preparation instructions accordingly? Thank you!

Sorry about that, it should be fixed now!

**Review by hannahbaumann — APPROVED** [review]
Thank you @tlhr , this looks good to me!

**Review by IAlibay — APPROVED** [review]
Thanks @tlhr !

**Comment by tlhr** [comment]
Just adding some minor comments here for completeness:
* Downgraded to `cudatoolkit=11.4` for cluster compatibility
* Ran using Nvidia multi-process-service (MPS), by adding the following lines to the job submission script (our GPUs are in `Exclusive Process` mode by default, which is not compatible with `openmmtools`):
```bash
export MPS_DIR=$(mktemp -d)
export CUDA_MPS_LOG_DIRECTORY=$MPS_DIR
export CUDA_MPS_PIPE_DIRECTORY=$MPS_DIR
nvidia-cuda-mps-control -d
```

**Comment by IAlibay** [comment]
> Just adding some minor comments here for completeness:
> 
> * Downgraded to `cudatoolkit=11.4` for cluster compatibility
> * Ran using Nvidia multi-process-service (MPS), by adding the following lines to the job submission script (our GPUs are in `Exclusive Process` mode by default, which is not compatible with `openmmtools`):
> 
> ```shell
> export MPS_DIR=$(mktemp -d)
> export CUDA_MPS_LOG_DIRECTORY=$MPS_DIR
> export CUDA_MPS_PIPE_DIRECTORY=$MPS_DIR
> nvidia-cuda-mps-control -d
> ```

O that's really interesting, I didn't know you could leverage MPS this way, thanks so much for sharing @tlhr ! We would really be interested in any performance number you get.

**Comment by tlhr** [comment]
> O that's really interesting, I didn't know you could leverage MPS this way, thanks so much for sharing @tlhr ! We would really be interested in any performance number you get.

Sure! `simulation_real_time_analysis.yaml` is reporting ~300 ns/day on a single V100 (This must be the aggregated number over all replicas, so around 30 ns/day/replica).

**Pull request merged**